packages/params/src/presets/minimal/phase0.ts

```diff
@@ -38,7 +38,7 @@ export const phase0: Phase0Preset = {
   // 2**0 (= 1) slots 6 seconds
   MIN_ATTESTATION_INCLUSION_DELAY: 1,
   // [customized] fast epochs
-  SLOTS_PER_EPOCH: 8,
+  SLOTS_PER_EPOCH: 3,
```

Comment:
Don't commit this. Set this way to match https://github.com/Inphi/eip4844-interop/blob/master/prysm/prysm-chain-config.yml#L6